src/test/java/com/compilercharisma/chameleonbusinessstudio/dto/ScheduleValidatorTester.java

```diff
@@ -0,0 +1,71 @@
+package com.compilercharisma.chameleonbusinessstudio.dto;
```

Comment:
/Very minor: I think the package name for these tests should be named `./validator/...` not `./dto/...`